tests/util.py

```diff
@@ -69,3 +79,55 @@ def xdist_sort_hack(it: Iterable[_R]) -> Iterable[_R]:
 
 def actions_output_to_dict(output: str) -> dict[str, str]:
     return {line.split("=")[0]: line.split("=")[1] for line in output.splitlines()}
+
+
+def get_release_history_from_context(runtime_context: RuntimeContext) -> ReleaseHistory:
+    rh = ReleaseHistory.from_git_history(
+        runtime_context.repo,
+        runtime_context.version_translator,
+        runtime_context.commit_parser,
+        runtime_context.changelog_excluded_commit_patterns,
+    )
+    changelog_context = make_changelog_context(runtime_context.hvcs_client, rh)
+    changelog_context.bind_to_environment(runtime_context.template_environment)
+    return rh
+
+
+def prepare_mocked_git_command_wrapper_type(
```

Comment:
This is sorcery but the useful kind :+1: I remember not really figuring out how to test the command line without adding `--no-push` so I can see this really helping some of those tests too